### DISCUSSION TITLE
I don't know how to pass data to the calendar

### DISCUSSION THREAD

**Discussion by akakJnr** [nhn/tui.calendar#1341] [Q&A] [answered]
I don't know where the data is coming from into the calendar UI.
 
Please how do I pass data to the calendar, thanks.

**Comment by adhrinae** [comment, marked as answer]
You need to manage the data on your own.

https://github.com/nhn/tui.calendar/blob/main/docs/en/guide/getting-started.md#creating-events

**Reply by akakJnr** [reply]
I added the create event method, but it doesn't work

**Reply by adhrinae** [reply]
Please describe the situation and upload a sample code or sandbox(jsfiddle, codesandbox, etc.) to find out what went wrong.
